vscode-antimony/src/extension.ts

```diff
@@ -35,6 +35,17 @@ let activeEditor = vscode.window.activeTextEditor;
 // RoundTripping SBML to Antimony
 let roundTripping: boolean | null = null;
 
+// Check if the current file is .txt
+async function checkFileExtension() {
+	const doc = vscode.window.activeTextEditor.document;
+	const uri = doc.uri.toString();
+	const fileExtension = path.extname(uri);
+	if (fileExtension === '.txt') {
+		vscode.window.showErrorMessage('Please save the file as .ant to use Vscode-Antimony', {modal: true});

```

Comment:
There are cases when user uses a txt file for non antimony purposes, so the message might be confusing